types/leaflet.sync/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for Leaflet.Sync 0.2
+// Project: https://github.com/jieter/Leaflet.Sync
+// Definitions by: Gabriel O. Martins <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 5.0
```

Comment:
```suggestion
```
You dont need this.

Reply:
without it i get the following error while running the tests:
> definitely-typed@0.0.3 test
> node --require source-map-support/register node_modules/@definitelytyped/dtslint/ types leaflet.sync

dtslint@0.0.162
Error: Cannot find module 'C:\Users\gabriel.martins\.dts\typescript-installs\4.3\node_modules\typescript'
Require stack:
- C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\dist\lint.js
- C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\dist\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1082:15)
    at Function.Module._load (node:internal/modules/cjs/loader:928:27)
    at Module.require (node:internal/modules/cjs/loader:1149:19)
    at require (node:internal/modules/helpers:121:18)
    at testDependencies (C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\src\lint.ts:94:29)
    at lint (C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\src\lint.ts:30:20)
    at testTypesVersion (C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\src\index.ts:233:15)
    at runTests (C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\src\index.ts:197:7)
    at main (C:\Users\gabriel.martins\Desktop\dev\DefinitelyTyped\node_modules\@definitelytyped\dtslint\src\index.ts:83:5)